benchmark/scenarios/request_load.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Read request load scenario for the control service benchmarks.
+"""
+from itertools import repeat
+
+from zope.interface import implementer
+from eliot import start_action, write_failure, Message
+from eliot.twisted import DeferredContext
+
+from twisted.internet.defer import CancelledError, Deferred
+from twisted.internet.task import LoopingCall
+
+from flocker.common import loop_until, timeout
+
+from .._interfaces import IScenario
+
+from rate_measurer import RateMeasurer, DEFAULT_SAMPLE_SIZE
+
+
+class RequestRateTooLow(Exception):
+    """
+    The request rate dropped below a threshold.
+    """
+
+
+class RequestRateNotReached(Exception):
+    """
+    The request rate did not reach the target level.
+    """
+
+
+class RequestOverload(Exception):
+    """
+    There are too many outstanding requests.
+    """
+
+
+class NoNodesFound(Exception):
+    """
+    No nodes were provided by the control service.
+    """
+
+
+@implementer(IScenario)
+class RequestLoadScenario(object):
```

Comment:
Make this a `PClass`, since there is no modification of the instance fields.  List the docstring `ivar`s in the same order as the PClass fields.